### PR TITLE
Use `ServiceMetadata` strings in Helm charts

### PR DESCRIPTION
**Pull request by RedbackThomson**
Description of changes:
Use the fields in the service metadata file (`metadata.yaml`) to produce nicer descriptions in the Helm chart. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/metadata/service_metadata.go [hunk 1]
```diff
@@ -104,17 +104,17 @@ func (m *ServiceMetadata) getVersionsByStatus(status APIStatus) []string {
 // path to a metadata file
 func NewServiceMetadata(
 	metadataPath string,
-) (ServiceMetadata, error) {
+) (*ServiceMetadata, error) {
```
Comment (A-Hilaly) [inline pkg/metadata/service_metadata.go:107]: ++

**Review by A-Hilaly — APPROVED** [review]
Neat 👍

**Review by jaypipes — APPROVED** [review]
Nice improvement. ++

**Pull request merged**